test/test_overrides.py

```diff
@@ -0,0 +1,756 @@
+import torch
```

Comment:
I think you need to add this file to `run_test.py`

Reply:
cc @mruberry do you think you could take a look at this file?

Reply:
Yep.